mod_reforged/hooks/config/strings.nut

```diff
@@ -2144,7 +2153,7 @@ foreach (vanillaDesc in vanillaDescriptions)
 			{
 				Type = ::UPD.EffectType.Passive,
 				Description = [
-					"Every hit or miss against any target increases your [Initiative|Concept.Initiative] by " + ::MSU.Text.colorPositive("+15") + ". This bonus lasts over into your next [turn|Concept.Turn] but only until the first skill used or waiting that [turn.|Concept.Turn]",
+					"Every hit or miss against any target increases your [Initiative|Concept.Initiative] by " + ::MSU.Text.colorPositive("+15") + ". This bonus lasts over into your next [turn|Concept.Turn] but only until the first skill used or upon waiting that [turn.|Concept.Turn]",
```

Comment:
- `lasts over` could be replaced with `is carried over`
- waiting should be hyperlinked: `[waiting|Concept.Wait]`